life-and-letters/original/02-chapter-2.adoc

```diff
@@ -120,54 +128,65 @@ and whilst it would be unreasonable to expect to see them, in every respect,
 such as would now be framed, it will be well, in judging of them,
 to reflect what dignified characters were trained under their strict administration.
 
-Rules of Conduct for Girls at School.
+[.alt.centered]
+=== Rules of Conduct for Girls at School.
 
-1st. Never absent yourselves from school, unless good reasons can be assigned,
+[.numbered]
+__1st.__ Never absent yourselves from school, unless good reasons can be assigned,
 to the satisfaction of your mistress.
 
-2nd. Be always silent at your business, so that your voices shall not be heard,
+[.numbered]
+__2nd.__ Be always silent at your business, so that your voices shall not be heard,
 unless when you are saying your lessons, or speaking to your mistress;
 hold no discourse with your schoolmates during the hours of business,
 without your mistress's permission, unless to ask something relating to your learning,
 which must be done in a low voice.
 
-3rd. Make all your speeches to your mistress with due respect,
+[.numbered]
+__3rd.__ Make all your speeches to your mistress with due respect,
 observing cheerfully to perform her directions with dispatch, according to your ability.
 If a stranger should speak to you, give a modest and ready answer,
 standing up and turning your faces towards them respectfully, taking your seats again,
 and silently apply to your business.
 
-4th. Behave yourselves always in a humble, obliging manner to your schoolmates,
+[.numbered]
+__4th.__ Behave yourselves always in a humble, obliging manner to your schoolmates,
 never provoking one another, contending or complaining about frivolous matters,
 but courteously use the word please, or some soft and obliging expression,
 when you ask anything one of the other;
 and observe to make some grateful return for any little kindness received,
 never returning injuries, but learning to forgive,
 showing by an exemplary deportment how all ought to behave.
 
-5th. Never tell a lie knowingly, nor use the name of God irreverently, or in vain;
+[.numbered]
+__5th.__ Never tell a lie knowingly, nor use the name of God irreverently, or in vain;
 mock not the aged, blind, lame, deformed, idiots.
 Play not in the streets, or ever for gain; throw no sticks, stones, dirt, snowballs,
 or anything at any person; revile none, utter no indecent expression,
 and return no affront to any you may meet with,
 but according to the best of your knowledge, demean yourselves in a modest, civil,
 complaisant manner to all.
 
-6th. Let your language be the plain language, and spoken with propriety everywhere.
+[.numbered]
+__6th.__ Let your language be the plain language, and spoken with propriety everywhere.
 
-7th. Be not forward to divulge anything past in school,
+[.numbered]
+__7th.__ Be not forward to divulge anything past in school,
 nor jeer those who have been corrected, lest it should happen to be your own case.
 
-At Home.
+[.alt.centered]
+=== At Home.
```

Comment:
I think you only updated the headings in the _modernized_ edition, probably should do the same in _original_ as well.

Reply:
That was dumb.